### PR TITLE
MWC4K 2021 news post/wiki article

### PR DESCRIPTION
**Pull request by LeoFLT**
Time to smash some keyboards.

Added a news post and a wiki article file, along with the banners used in them. I also made some updates to the tournament index page (add CWC2021 results/MWC 4K 2021 to the listing).

### CODE REVIEW

File: news/2021-07-07-osumania-4k-world-cup-2021-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+*Notice: you may only participate if your osu!mania 4K Global Ranking is #5000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months*.
+
+If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.
+
+*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of schedule to avoid any last-minute issues.*
+
+## New leadership
+
+Each World Cup comes with new and exciting things, and this time around will be no exception. With our previous organisers juankristal and WalterToro stepping down it leaves some large shoes to fill. Starting from today, ChillierPear, LeoFLT, and Azer will be stepping up to meet the challenges and expectations as the new organisers of the osu! World Cups.
```
Comment (Walavouchey) [inline news/2021-07-07-osumania-4k-world-cup-2021-registrations-now-open.md:47]:
```suggestion
Each World Cup comes with new and exciting things, and this time around will be no exception. With our previous organisers [juankristal](https://osu.ppy.sh/users/443656) and [WalterToro](https://osu.ppy.sh/users/5281416) stepping down, large shoes are to be filled. Starting from today, [ChillierPear](https://osu.ppy.sh/users/9501251), [LeoFLT](https://osu.ppy.sh/users/3668779), and [Azer](https://osu.ppy.sh/users/2155578) will be stepping up to meet the challenges and expectations as the new organisers of the osu! World Cups.
```

File: news/2021-07-07-osumania-4k-world-cup-2021-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+"The three of us are very excited for this opportunity. In recent years, most, if not all of our time on osu! has been dedicated towards developing and nurturing the tournament scene. The osu! World Cups will still continue as they have been, and we hope to take them in a forward direction through the coming years and continue to provide the consistently high quality and entertaining experience that has existed for so long under the direction of our predecessors."
```
Comment (Walavouchey) [inline news/2021-07-07-osumania-4k-world-cup-2021-registrations-now-open.md:49]:
```suggestion
> The three of us are very excited for this opportunity. In recent years, most, if not all of our time on osu! has been dedicated towards developing and nurturing the tournament scene. The osu! World Cups will still continue as they have been, and we hope to take them in a forward direction through the coming years and continue to provide the consistently high quality and entertaining experience that has existed for so long under the direction of our predecessors.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+| :-- | :-- |
+| General Manager | ![][flag_CA] [Azer](https://osu.ppy.sh/users/2155578), ![][flag_US] [ChillierPear](https://osu.ppy.sh/users/9501251), ![][flag_SE] [Davvy](https://osu.ppy.sh/users/10047413), ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779) |
+| Mappool selector | ![][flag_US] [-mint-](https://osu.ppy.sh/users/8976576), ![][flag_SE] [Davvy](https://osu.ppy.sh/users/10047413), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) |
+| Referee | ![][flag_AU] [Fairy Bread](https://osu.ppy.sh/users/8306102), ![][flag_US] [JDrago14](https://osu.ppy.sh/users/7690078), ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_NL] [nik](https://osu.ppy.sh/users/10077264), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Commentator | TBD |
+| Statistician | ![][flag_BR] [LeoFLT](https://osu.ppy.sh/users/3668779), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Signup here](https://osu.ppy.sh/community/tournaments/29)
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:51]:
```suggestion
- [Sign up here](https://osu.ppy.sh/community/tournaments/29)
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+3. The beatmaps for each round will be announced by the Map Selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
+   - One beatmap will be given as a tiebreaker beatmap. This beatmap will only be played in case of a tie.
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players' scores do not get added to the team score.
+   - Reviving and surviving during a beatmap is considered as passing it.
+7. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
+8. If the beatmap ends in a draw, the game will be nullified.
+9. If a player disconnects, they get treated as if they failed the beatmap.
+   - Disconnects within 30 seconds after beatmap begin can be rematched. This is up to the referee's discretion. To this end, the lobby is subject to the !mp abort command.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:71]:
```suggestion
   - Disconnects within 30 seconds after starting can be rematched. This is up to the referee's discretion. To this end, the lobby is subject to the `!mp abort` command.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+   - If a rematch happens, the roster for each team during that particular beatmap must remain the same. If that is not possible, i.e., if the player who disconnected can not get back in time, *both* teams will be allowed to swap rosters.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:72]:
```suggestion
   - If a rematch happens, the roster for each team during that particular beatmap must remain the same. If that is not possible, i.e. if the player who disconnected cannot get back in time, *both* teams will be allowed to swap rosters.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+10. Beatmaps cannot be reused in the same match unless the game has been nullified.
+11. **The maximum size for a team is 6 players, and the minimum is 3.**
+    - This might change depending on the amount of registrations.
+12. If less than the minimum required players attend, the maximum time the match can be postponed is 10 minutes. A 'Win by default' will be declared if there isn't enough players after this deadline.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:76]:
```suggestion
12. If less than the minimum required players attend, the maximum time the match can be postponed is 10 minutes. A *win by default* will be declared if there aren't enough players after this deadline.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+13. Exchanging players during a match is allowed without limitations.
+14. Lag is not a valid reason to nullify a beatmap.
+15. All players will be expected to keep the match running fluently and without delays. Excessive match delays from the players' side can be issued with penalties.
+16. If a player disconnects between beatmaps, and the team can not provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match).
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:80]:
```suggestion
16. If a player disconnects between beatmaps, and the team cannot provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match).
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+17. All players and staff must be treated with respect. Instructions of the referees and the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
+18. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match or other deliberate inappropriate misbehavior are strictly prohibited.
+19. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:84]:
```suggestion
    - Breaking the chat rules results in a silence. Silenced players cannot participate at multiplayer matches and must be exchanged for the time being.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+11. **The maximum size for a team is 6 players, and the minimum is 3.**
+    - This might change depending on the amount of registrations.
+12. If less than the minimum required players attend, the maximum time the match can be postponed is 10 minutes. A 'Win by default' will be declared if there isn't enough players after this deadline.
+13. Exchanging players during a match is allowed without limitations.
+14. Lag is not a valid reason to nullify a beatmap.
+15. All players will be expected to keep the match running fluently and without delays. Excessive match delays from the players' side can be issued with penalties.
+16. If a player disconnects between beatmaps, and the team can not provide a substitute, the match can be delayed for up to 10 minutes (limited to once per team, per match).
+17. All players and staff must be treated with respect. Instructions of the referees and the Tournament Management are to be followed. Decisions labeled as final are not to be objected.
+18. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match or other deliberate inappropriate misbehavior are strictly prohibited.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:82]:
```suggestion
18. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match or other deliberate inappropriate misbehaviour are strictly prohibited.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 10]
```diff
@@ -0,0 +1,195 @@
...
+19. The multiplayer chatrooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chatrooms, too.
+    - Breaking the chat rules results in a silence. Silenced players can not participate at multiplayer matches and must be exchanged for the time being.
+20. Unexpected incidences will be handled by the Tournament Management. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+21. Penalties for violating the tournament rules can be:
+    - Exclusion of specific players for one beatmap.
+    - Exclusion of specific players for an entire match.
+    - Declaring the match as "Lost by Default".
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:89]:
```suggestion
    - Declaring the match as lost by default.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+    - Disqualification from the entire tournament.
+    - Disqualification from the current and future official tournaments until appealed.
+    - Any modification of these rules will be announced.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:92]: this is not a penalty is it?

```suggestion
22. Any modification of these rules will be announced.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 12]
```diff
@@ -0,0 +1,195 @@
...
+
+### Tournament registration
+
+1. Every user interested in joining their country's team signs up individually.
+   - Tournament Management will create a list of potential candidates for a country's team.
+   - Tournament Management declares one candidate to the captain of the country's team, albeit temporarily.
+   - The declared captain can form their team from the candidate list of their country.
+2. To ensure valid and serious registrations, every registered user will be checked by the Tournament Management.
+   - Every registered user will be assigned to their respective country's candidate list.
+   - To be successfully accepted on the list, you have to ensure that your global osu!mania 4K performance ranking is above 5000.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:102]:
```suggestion
   - To be successfully accepted on the list, you have to ensure that your global osu!mania 4K performance ranking is above #5000.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 13]
```diff
@@ -0,0 +1,195 @@
...
+   - To be successfully accepted on the list, you have to ensure that you did not violate the [osu! community rules](/wiki/Rules) within the last 12 months.
+3. All successfully formed teams will be published after the Registration Phase.
+4. Only the 32 potentially strongest countries will participate. The potential strength of a country is determined by the online statistics of all valid candidates.
+5. Mapset selectors must not participate as a player in this tournament.
+
+### Stage instructions
+
+1. In the Qualifier stage, all teams will play a specific pool designed by the map selectors.
+2. The pool will contain 4 brackets: NoMod, [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), [DoubleTime](/wiki/Game_modifier/Double_Time). There will not be a tiebreaker for this stage.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:111]:
```suggestion
2. The pool will contain 4 brackets: NoMod, [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), and [DoubleTime](/wiki/Game_modifier/Double_Time). There will not be a tiebreaker for this stage.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 14]
```diff
@@ -0,0 +1,195 @@
...
+3. The Qualifier pool contains 10 maps. There will be 4 maps under the NoMod bracket and 2 maps for the other mod brackets.
+4. Teams will have to play the map pool twice at a designated time. Their best combined score will be used for seeding.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:113]:
```suggestion
4. Teams will have to play the mappool twice at a designated time. Their best combined score will be used for seeding.
```

File: wiki/Tournaments/MWC/2021_4K/en.md [hunk 15]
```diff
@@ -0,0 +1,195 @@
...
+5. The mappool will be played in the order listed above.
+6. Each team must have 4 players for each map. They can be exchanged freely after a map is concluded.
+7. There will be a 5-minute break in between the first and second play-through of the map pool.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2021_4K/en.md:116]:
```suggestion
7. There will be a 5-minute break in between the first and second play-through of the mappool.
```

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**